tests/integration/targets/vmware_guest_instant_clone/tasks/main.yml

```
@@ -0,0 +1,387 @@
+# Test code for the vmware_guest_instant_clone Operations.
+# Copyright: (c) 2021, Anant Chopra <chopraan@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_datacenter: true
```

```suggestion
    setup_datacenter: true
    setup_resource_pool: true
```

By setup_resource_pool is true, you can create a resource pool to a real lab of CI.
https://github.com/ansible-collections/community.vmware/blob/main/tests/integration/targets/prepare_vmware_tests/tasks/setup_resource_pool.yml